### DISCUSSION TITLE
请问硕士论文一级标题英文字体怎么修改

### DISCUSSION THREAD

**Discussion by stellalaland** [note286/xduts#197] [Q&A] [answered]
硕士模版目录以及正文里的一级标题英文字体是黑体，怎么改成new roman呢🙏🏻

**Comment by note286** [comment, marked as answer]
首先要说明，无衬线中文字体和衬线英文字体混用是不合适的，目前xdtus的默认设置没有问题。

因此，并不建议进行你所述的设置，不过，仍然满足你，下面给出代码。

如果仅仅设置正文一级标题样式而目录不修改：

```latex
\documentclass{xdupgthesis}
\AtBeginDocument{\ctexset{chapter/format+=\CJKfamily{sf}\CJKfamily-{rm}}}
\begin{document}
\chapter{测试test}
\end{document}
```

![image](https://github.com/user-attachments/assets/69fadad0-19ab-42d6-9cf0-4badc0ab9980)


如果正文一级标题和目录均修改：

```latex
\documentclass{xdupgthesis}
\AtBeginDocument{\ctexset{chapter/format+=\CJKfamily{sf}\CJKfamily-{rm}}}
\AtBeginDocument{\let\oldcftchapfont\cftchapfont\renewcommand{\cftchapfont}{\oldcftchapfont\CJKfamily{sf}\CJKfamily-{rm}}}
\begin{document}
\chapter{测试test}
\end{document}
```

![image](https://github.com/user-attachments/assets/3d24a33e-3951-4dda-a6cd-181a5fa83bdb)
